crates/bevy_pbr/src/render/pbr_functions.wgsl

```diff
@@ -254,7 +254,7 @@ fn pbr(
 
     // Total light
     output_color = vec4<f32>(
-        direct_light + indirect_light + emissive_light,
+        view.exposure * (direct_light + indirect_light + emissive_light),
```

Comment:
This assumes that indirect light is already in photometric units. There may need to be a scale factor to undo any pre-exposure that's baked into the contents of the cubemaps.

Reply:
Yeah. There was talk about an "intensity" field for environment map lighting. I decided to leave it out in my original PR because I was worried it would be abused. Maybe it's time for that now.